backend/development.ini.sample

```diff
@@ -338,23 +356,26 @@ website.base_url = %(basic_setup.website_base_url)s
 # reply+{content_id}{content_id}@trac.im
 
 # example of recommanded pattern (for 'reply_to' header):
-; email.notification.reply_to.email = reply+{content_id}@domainname.tld
-; email.notification.reply_to.email = {content_id}@reply.domainname.tld
-; email.notification.reply_to.email = reply+cid{content_id}@domainname.tld
+# email.notification.reply_to.email = reply+{content_id}@domainname.tld
+# email.notification.reply_to.email = {content_id}@reply.domainname.tld
+# email.notification.reply_to.email = reply+cid{content_id}@domainname.tld
 
-## valid variable is 'user_id' for 'from' header
-; email.notification.from.email = noreply+{user_id}@trac.im
+## valid variable is 'user_id' for 'from' header:
+# email.notification.from.email = noreply+{user_id}@trac.im
 
 # tagging reply_to header of email with content_id is needed for working
 # email_reply feature
-## valid variable is 'content_id' for 'reply_to' header
-; email.notification.reply_to.email = reply+{content_id}@trac.im
+## valid variable is 'content_id' for 'reply_to' header:
+# email.notification.reply_to.email = reply+{content_id}@trac.im
+
 # tagging reference header of email with content_id is nice to have
 # all modification to a same content in a same email client thread
 # Note: reference doesn't need to be an existing email account but should
 # be a email like string.
 ## valid variable is 'content_id' for 'references' header
-; email.notification.references.email = thread+{content_id}@trac.im
+# email.notification.references.email = thread+{content_id}@trac.im
+
```

Comment:
I think for here, we have exemple in beginning so we need empty value and not `# at beginning of each necessary line`:
- `## valid variable is 'user_id' for 'from' header:`
- `; email.notification.from.email = `
- `## valid variable is 'content_id' for 'reply_to' header:`
- `; email.notification.reply_to.email = `
- `## valid variable is 'content_id' for 'references' header:`
- `; email.notification.references.email = `

Reply:
fixed